README.rst

```diff
@@ -1,5 +1,5 @@
 ====================
-caishen-dashboard-BE
+caishen-dashboard
```

Comment:
I have gone over these [in this PR](https://github.com/Kerem-Sami-Coop/caishen-dashboard/pull/4) as well could you go over them